.travis.yml

```diff
@@ -87,7 +87,9 @@ before_install:
   - conda update -q conda
   # Useful for debugging any issues with conda
   - conda info -a
-  - sudo apt-get install -y -q mpich libmpich-dev
+  - if [[ "$MPI_INSTALL" == '1' ]]; then
+      sudo apt-get install -y -q mpich libmpich-dev;
```

Comment:
BTW, now that I look at this -- we should **not** use sudo here when we have a conda environment